doc/02-Installation.md

```diff
@@ -60,6 +63,9 @@ in `Configuration / Application`. In a typical environment you'll probably drop
 module to `/usr/share/icingaweb2/modules/director`. Please note that the directory
 name MUST be `director` and not `icingaweb2-module-director` or anything else.
 
```

Comment:
I intentionally did not propose a specific installation method, git VS release tar etc. Installing modules is something that is explained elsewhere and not something someone starting with Director should still have to learn about. Please either remove this or replace it with a larger section explaining the pros and cons of the different installation methods. Both variants are fine to me.